pandas/tests/frame/methods/test_to_csv.py

```diff
@@ -759,9 +759,11 @@ def test_to_csv_chunking(self, chunksize):
             tm.assert_frame_equal(rs, aa)
 
     @pytest.mark.slow
-    def test_to_csv_wide_frame_formatting(self):
+    def test_to_csv_wide_frame_formatting(self, monkeypatch):
```

Comment:
this is a specific regression test https://github.com/pandas-dev/pandas/issues/8621

i dont' think changing this is actually helping.